src/coverlet.console/Program.cs

```diff
@@ -15,6 +15,42 @@ namespace Coverlet.Console
 {
     class Program
     {
+        /// <summary>
+        /// Exit Codes returned from Coverlet console process.
+        /// </summary>
+        enum CommandExitCodes
```

Comment:
I like the idea of enum, but I prefer to have a custom file `ExitCodes.cs` with internal enum inside, the "real program logic" is clearer(this is only a struct).

Reply:
Since discussion was to create only for console application, I didn't abstract much. Should I create `ExitCodes.cs` in Coverlet.Core project? or still in Coverlet.Console project?

Reply:
`Coverlet.Console`